src/AdminMenus/FormListingAdminSubMenu.php

```diff
@@ -56,6 +59,8 @@ function () {
 			},
 			20
 		);
+
+		\add_action('admin_menu', [$this, 'addCustomLinkIntoAppearnaceMenu'], 32);
```

Comment:
```suggestion
		\add_action('admin_menu', [$this, 'addCustomLinkIntoAppearanceMenu'], 32);
```

Reply:
Suggestion to the suggestion: `addCustomLinkToAppearanceMenu`.